Makefile

```diff
@@ -0,0 +1,53 @@
+PYTHON ?= $(shell command -v python3 2>/dev/null || command -v python)
+DESTDIR ?= /
+PODMAN_VERSION ?= '1.80'
+
+.PHONY: podman-py
+podman-py:
+	PODMAN_VERSION=$(PODMAN_VERSION) \
+	$(PYTHON) setup.py sdist bdist
+
+.PHONY: env
+env:
+	dnf install python3-coverage
```

Comment:
Should we add a `-y` to these

Reply:
Also should we support apt versions for this, probably do this in a different PR.

Reply:
im actually wondering if we should detect the presense of python3-coverage and bail if not?

Reply:
I'm very much against installing system-wide stuff from a makefile.  It presumes a lot about the environment it's running under, and that the developer even wants their system messed with for them :grinning:

A better approach would be to stick this stuff into a shell script.  That can be used by both humans and machines as needed.

Reply:
@rhatdan Done
@baude Done
@cevich Note that target has no dependencies so it will only run if called.